File: leetcode/problem0001.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let indicies = []
    for(let i = 0; i < nums.length; i++){
        for(let j = i + 1; j < nums.length; j++){
            if(nums[i] + nums[j] === target) {
                indicies.push(i, j)
            }
        }

    }
    return indicies
};

// https://leetcode.com/problems/two-sum/

File: leetcode/problem0001.go
func twoSum(nums []int, target int) []int {
    var solution []int
    for i := 0; i < len(nums); i++ {
        for j := i + 1; j < len(nums); j++ {
            if nums[i] + nums[j] == target {
                solution = append(solution, i, j)
            }
        }
    }
    return solution
}

// https://leetcode.com/problems/two-sum/
